src/tabs/Pixnflix/index.tsx

```diff
@@ -195,36 +243,16 @@ class PixNFlix extends React.Component<Props, State> {
     }
   };
 
-  public printError: ErrorLogger = () => {};
-
-  private swapModes = (mode: SideContentVideoDisplayMode) => () => {
-    switch (mode) {
-      case 'video':
-        this.setState(
-          (state: State) => ({
-            ...state,
-            mode: 'still' as SideContentVideoDisplayMode,
-          }),
-          this.handleSnapPicture
-        );
-        break;
-      // case 'still':
-      default:
-        this.setState(
-          (state: State) => ({
-            ...state,
-            mode: 'video' as SideContentVideoDisplayMode,
-          }),
-          this.handleStartVideo
-        );
-        break;
-    }
+  public handleVolumeChange = (e: ChangeEvent<HTMLInputElement>) => {
+    e.preventDefault();
+    const volume = parseFloat(e.target.value);
+    this.setState({
+      volume,
+    });
+    this.pixNFlix.updateVolume(volume);
   };
 
-  private handleSwapModes: () => void = () => {
-    const { mode } = this.state;
-    this.swapModes(mode)();
-  };
+  public printError: ErrorLogger = () => {};

```

Comment:
Is it fitting to scrap the `ErrorLogger` system, here and over on the `functions.ts`/`init()` end?